vue-components/tests/unit/components/TextArea.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { mount } from '@vue/test-utils';
+import TextArea from '@/components/TextArea.vue';
+import { ResizeLimit } from '@/components/ResizeLimit.ts';
+
+describe( 'TextArea.vue', () => {
+	it( 'accepts rows property', () => {
+		const wrapper = mount( TextArea, {
+			propsData: { rows: 42 },
+		} );
+
+		expect( wrapper.props().rows ).toBe( 42 );
+		expect( wrapper.find( 'textarea' ).attributes( 'rows' ) ).toBe( '42' );
+	} );
+
+	it( 'accepts resize property', () => {
+		const wrapper = mount( TextArea, {
+			propsData: { resize: ResizeLimit.Horizontal },
+		} );
+
+		expect( wrapper.props().resize ).toBe( ResizeLimit.Horizontal );
+		expect( wrapper.find( 'textarea' ).classes() ).toContain( 'wikit-TextArea__textarea--horizontal' );
+	} );
+
```

Comment:
```suggestion
	it( 'uses default resize value', () => {
		const wrapper = mount( TextArea, { } );

		expect( wrapper.find( 'textarea' ).classes() ).toContain( 'wikit-TextArea__textarea--vertical' );
	} );
```
(Don't know if that can be run anonymously without the `const wrapper`.)

Reply:
Nice, this is a good way to mitigate the lost test.